config_src/drivers/nuopc_cap/mom_cap.F90

```diff
@@ -40,8 +30,7 @@ module MOM_cap_mod
 use MOM_cap_methods,          only: ChkErr
 
 #ifdef CESMCOUPLED
```

Comment:
Can't this `#ifdef` be replaced with a more _elegant_ `if (cesm_coupled)` that seems to work elsewhere in this module?

Reply:
`shr_log_mod` module is available in CESM only. So, unfortunately, an `ifdef` is the only option here.

Reply:
@sanAkel Did you want to follow up?  We cannot merge until all discussions have been resolved, even if the PR has been approved.

Note that driver maintainers are largely self-maintained and generally have final say on the code.

Reply:
@sanAkel I am going to close this, since the PR was approved.

There's probably a useful discussion here about when and how to preprocess, but wen can defer it to some other time.

Reply:
@marshallward, based on @alperaltuntas's answer, no objections to merging this PR.